sdk/python/feast/feature_server.py

```diff
@@ -0,0 +1,57 @@
+import uvicorn
+from fastapi import FastAPI, HTTPException, Request
+from fastapi.logger import logger
+from google.protobuf.json_format import MessageToDict, Parse
+
+import feast
+from feast.protos.feast.serving.ServingService_pb2 import GetOnlineFeaturesRequest
+from feast.type_map import feast_value_type_to_python_type
+
+
+def get_app(store: "feast.FeatureStore"):
```

Comment:
Why is the type annotation `"feast.FeatureStore"` instead of just FeatureStore?

Reply:
This is to avoid cyclical imports. FeatureStore already imports feature_server.py (in order to run start_server), while the feature_server.py doesn't really need to import FeatureStore except for the type hinting.